sway-fmt-v2/src/fmt.rs

```diff
@@ -475,6 +474,88 @@ trait CompSciStudent: Programmer + Student {
             Formatter::format(&mut formatter, Arc::from(sway_code_to_format), None).unwrap();
         assert_eq!(correct_sway_code, formatted_sway_code)
     }
+    #[test]
+    fn test_method_calls() {
+        let sway_code_to_format = r#"script;
+
+struct Opts {
+    gas: u64,
+    coins: u64,
+    id: ContractId,
+}
+
+fn  main(       ) -> bool{
+    let default_gas  = 1_000_000_000_000           ;let fuelcoin_id = ~ContractId::from(0x018f59fe434b323a5054e7bb41de983f4926a3c5d3e4e1f9f33b5f0f0e611889);
+
+    let balance_test_id = ~ContractId :: from( 0x597e5ddb1a6bec92a96a73e4f0bc6f6e3e7b21f5e03e1c812cd63cffac480463 ) ;
+
+    let fuel_coin = abi(    TestFuelCoin, fuelcoin_id.into(       ) ) ;               
+
+    assert(fuelcoin_balance == 0);
+
+    fuel_coin.mint        {
+        gas:             default_gas
+    }
+
+    (11);
+
+    fuelcoin_balance = balance_of(fuelcoin_id, fuelcoin_id);
+    assert( fuelcoin_balance   == 11 ) ;
+
+    fuel_coin.burn {
+        gas: default_gas
+    }
+    (7);
+
+    fuelcoin_balance = balance_of(fuelcoin_id, fuelcoin_id);
+    assert(fuelcoin_balance == 4);
+
+    fuel_coin.force_transfer {
+        gas: default_gas
+    }
+    (3, fuelcoin_id, balance_test_id);
+
+    fuelcoin_balance = balance_of(fuelcoin_id, fuelcoin_id);
+    let balance_test_contract_balance = balance_of(fuelcoin_id, balance_test_id);
+    assert(fuelcoin_balance == 1);
+    assert(balance_test_contract_balance == 3);
+
+    true
+}"#;
```

Comment:
This test is actually specifically what @mohammadfawaz gave me for formatting method calls, hence why it was used. But I can see that smaller tests would be better, since we may only end up breaking one specific thing like inline method calls.